src/AcceptanceTests/Shared/Publishing.cs

```diff
@@ -58,7 +59,10 @@ class Subscriber : EndpointConfigurationBuilder
     {
         public Subscriber()
         {
-            EndpointSetup<DefaultTestServer>(publisherMetadata: p => p.RegisterPublisherFor<MyEvent>(typeof(Publisher)));
+            EndpointSetup<DefaultTestServer>(c =>
+            {
+                c.DisableFeature<AutoSubscribe>();
+            }, publisherMetadata: p => p.RegisterPublisherFor<MyEvent>(typeof(Publisher)));
```

Comment:
Since this is the test endpoint, ie the AcceptanceTest project it seems unlikely that this could have any effect on ASQ which is the publisher side in this test?